core_backend/app/llm_call/utils.py

```diff
@@ -36,6 +36,24 @@ async def _ask_llm_async(
         api_base=litellm_endpoint,
         api_key=LITELLM_API_KEY,
         metadata={"generation_name": litellm_model},
+        safety_settings=[
```

Comment:
this now always gets sent regardless of which model the proxy routes to right? Could you add a comment in the code clarifying this is only for gemini maybe?

Reply:
Good idea thanks!

Reply:
OpenAI errors out if this is sent to it. Should move this to the proxy definition